objcgen/Makefile

```diff
@@ -15,3 +15,29 @@ ifeq ($(SHELLCHECK),)
 else
 	$(SHELLCHECK) $^
 endif
+
+VERSION = 0.1
```

Comment:
^ that's a good thing to generalize :)

but it would be better to copy paste the _git distance_ code from `xamarin-macios` so we don't end up shipping hundreds of different `0.1` packages